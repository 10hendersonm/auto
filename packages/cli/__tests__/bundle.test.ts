import path from "path";
import { execSync } from "child_process";

test("bundle should function", () => {
  const binary =
    (process.platform === "win32" && "win.exe") ||
    (process.platform === "darwin" && "macos") ||
    "linux";

  // Using this command because it is unlikely to change very much
  expect(
    execSync(
      path.join(__dirname, `../binary/auto-${binary} create-labels --help`),
      {
        encoding: "utf8",
      }
    )
  ).toMatchSnapshot();
});
