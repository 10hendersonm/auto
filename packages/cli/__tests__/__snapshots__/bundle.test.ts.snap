// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bundle should function 1`] = `
"
create-labels

  Create your project's labels on github. If labels exist it will update them. 

Options

  -d, --dry-run    Report what command will do but do not actually do anything 

Global Options

  -V, --version         Display auto's version                                  
  -v, --verbose         Show some more logs. Pass -vv for very verbose logs.    
  --repo string         The repo to set the status on. Defaults to looking in   
                        the package definition for the platform                 
  --owner string        The owner of the GitHub repo. Defaults to reading from  
                        the package definition for the platform                 
  --github-api string   GitHub API to use                                       
  --plugins string[]    Plugins to load auto with. (defaults to just npm)       
  -h, --help            Display the help output                                 

Examples

  $ auto create-labels 

"
`;
